pkg/devicemapper/devmapper.go

```diff
@@ -708,10 +710,15 @@ func DeleteDevice(poolName string, deviceID int) error {
 	}
 
 	dmSawBusy = false
+	dmSawEnoData = false
 	if err := task.run(); err != nil {
 		if dmSawBusy {
 			return ErrBusy
 		}
+		if dmSawEnoData {
+			logrus.Debugf("devicemapper: Device(id: %d) from pool(%s) do not exist", deviceID, poolName)
```

Comment:
s/do not exist/does not exist/

Reply:
It would be worth mentioning that the error is being ignored, like
```golang
+			logrus.Debugf("devicemapper: Device(id: %d) from pool(%s) does not exist, ignoring", deviceID, poolName)
￼ ```